app/views/cases/show.html.slim

```diff
@@ -11,7 +11,7 @@
 
 = render partial: 'shared/components/case_page_heading', locals: {kase: @case}
 
-div id="case-#{@correspondence_type_key}" class="case"
+div id="case-#{@correspondence_type_key == nil ? 'nil' : @correspondence_type_key.gsub('_', '-')}" class="case"
```

Comment:
This works great but there's a lot going on for one line in a view template. What do you think about maybe moving this out into a method in a decorator or a helper so we can name the concept and put a test around it?

Wondering where @correspondence_type_key is coming from, presumably one of the bazillion things SetupCase does or something like that?

Reply:
@Polsonby  this bug has been there for a while, just no one notice it until now, the change is nothing to do with any business logic, the issue I found is  if you put id='case-offender_sar', then somehow, it will try to append extra path to locate the css for table layout which I don't why 

IMO, this is purely for layout, it is not going to change anything and any value, I don't really feel it is valuable to add a test for testing whether this is a "visible" line there or not